src/lipalightninglib.udl

```diff
@@ -770,4 +764,5 @@ interface NotificationHandlingError {
 enum NotificationHandlingErrorCode {
     "NodeUnavailable",
     "InProgressSwapNotFound",
+    "ExpectedPaymentNotReceived",
```

Comment:
When looking at the implementation this error code makes a lot of sense, but from the PoV of the library consumer, I think this error code could be very confusing.

Wouldn't something like this be more telling? --> `CorruptPaymentNotification`

Also for the `InProgressSwapNotFound` --> `CorruptInProgressSwapNotification`

WDYT?

Reply:
I don't think the notifications were necessarily corrupt. `ExpectedPaymentNotReceived ` will likely be caused by an issue with the node or LSP that prevented the payment from being received. `InProgressSwapNotFound` may be caused by another client running concurrently and processing the swap first. 

I think it's better to keep error names descriptive of what occurs, but I may be missing the library consumer perspective. 

In the end, given how tightly coupled we are with the library consumer, either approach is fine. Let me know if you strongly prefer other names.